docs/data-sources/slos.md

```diff
@@ -199,3 +200,13 @@ Read-Only:
 Read-Only:
 
 - `query` (String)
+
+
+<a id="nestedobjatt--slos--query--ratio"></a>
+### Nested Schema for `slos.query.ratio`
+
+Read-Only:
```

Comment:
nit - Why does this say read-only? These are fields that user can edit in their terraform right?

I'm probably missing something, because a bunch of these sections include "read only" - forgive me if I'm unclear whether we expect others to consume this or the schema as documentation

Reply:
So this is the README for the SLOs Datasource - basically Terraform has two types of resources:
resources and datasources.
Datasources allow Terraform to read objects in - so that's why all the fields within this datasource markdown are read-only fields.